README.rst

```diff
@@ -114,6 +116,11 @@ Documentation can be generated in several formats::
     epub : make epub
     html : make html
 
+Contributing to libdrizzle-redux
+--------------------------------
+
+Please refer to the guide for users of libraries at `Neptune Wiki`_.
```

Comment:
So you mean that 

> What's Neptune about ? I am wondering where a function is used, is that where I should go ?

Should be the body of the section, followed by a link to neptune?